packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1147,11 +1185,16 @@ export default class Home extends Component {
               </div>
             )}
           </Col>
+
           <Col smallSize={6} mediumSize={2} className='text-xs-right'>
-            <Button onClick={this._expandAll}>
+            <Button onClick={this._expandAll} disabled={disableAction}>
               <Icon icon='nav' />
             </Button>{' '}
-            <DropdownButton bsStyle='info' title={homeItemsPerPage}>
+            <DropdownButton
+              bsStyle='info'
+              title={homeItemsPerPage}
+              disabled={disableAction}
```

Comment:
Why do you disable this?

Reply:
If it's checked, the action bar will be displayed and the `SortedTable` manage actions.

Reply:
But this is the dropdown to set the number of items per page. Shouldn't we still be able to set that?